src/workers/airtable-sync/ChangeDetector.js

```diff
@@ -0,0 +1,164 @@
+const _ = require("lodash");
+const { airbase, UPDATE_BATCH_SIZE } = require("../../airtable");
+
+// Maps airtable column names
+const metaField = "Meta";
+const lastModifiedField = "Last Modified";
+const lastProcessedField = "Last Processed";
+
+const overlap = 5 * 1000; // N second overlap for lastModified
```

Comment:
So we always do end up reading-back bookkeeping changes from Airtable but they get filtered out by `isModified` because the values in the row and the values in `meta.lastValues` match up. So they don't get their metadata updated again nor do they end up in the results. I'll add that to comments.

The overlap is to assuage the worry that there's jitter/clock skew in Airtable's `Last Modified` updates. There's no real guarantee or documentation around that stuff so we were guarding against something like:
  (1) change row X at 00:15 and `Last Modified` is set to 00:15
  (2) the thing polls at 00:16 and sees that row
  (3) change row X at 00:17 BUT there's some clock skew in Airtable's system so the `Last Modified` is set to 00:14. Now that change would never get processed.